File: tests/env_tester/champions/Champion.py
from api import *


# Function called at the start of the game
def init_game():
    # TODO
    pass


# Called when this is your turn to play
def play_turn():
    game_board = board()

    for i in range(3):
        for j in range(3):
            if game_board[3 * j + i] == -1:
                pos = (i, j)
                play(pos)
                return


# Function called at the end of the game
def end_game():
    # TODO
    pass

File: tests/env_tester/champions/champion.rs
// Include standard library in documentation: `cargo doc --open`
#[doc(inline)]
pub use std;

mod ffi;
pub mod api;

use api::*;


/// Function called at the start of the game
pub fn init_game()
{
    // TODO
}

/// Called when this is your turn to play
fn play_turn() {
    let game_board = board();

    for i in 0..3 {
        for j in 0..3 {
            if game_board[3 * j + i] == -1 {
                let pos = (i as i32, j as i32);
                play(pos);
                return;
            }
        }
    }
}

/// Function called at the end of the game
pub fn end_game()
{
    // TODO
}
